i18n/ko.json

```
@@ -1,1962 +1,1962 @@
-    "notification_email_ignore_certificate_errors": "Ignore certificate errors",
-    "notification_email_ignore_certificate_errors_description": "Ignore TLS certificate validation errors (not recommended)",
-    "notification_email_password_description": "Password to use when authenticating with the email server",
-    "notification_email_port_description": "Port of the email server (e.g 25, 465, or 587)",
-    "notification_email_sent_test_email_button": "Send test email and save",
-    "notification_email_setting_description": "Settings for sending email notifications",
-    "notification_email_test_email": "Send test email",
-    "notification_email_test_email_failed": "Failed to send test email, check your values",
-    "notification_email_test_email_sent": "A test email has been sent to {email}. Please check your inbox.",
-    "notification_email_username_description": "Username to use when authenticating with the email server",
-    "notification_enable_email_notifications": "Enable email notifications",
-    "notification_settings": "Notification Settings",
-    "notification_settings_description": "Manage notification settings, including email",
-    "oauth_auto_launch": "Auto launch",
-    "oauth_auto_launch_description": "Start the OAuth login flow automatically upon navigating to the login page",
-    "oauth_auto_register": "Auto register",
-    "oauth_auto_register_description": "Automatically register new users after signing in with OAuth",
-    "oauth_button_text": "Button text",
-    "oauth_client_secret_description": "Required if PKCE (Proof Key for Code Exchange) is not supported by the OAuth provider",
-    "oauth_enable_description": "Login with OAuth",
-    "oauth_mobile_redirect_uri": "Mobile redirect URI",
-    "oauth_mobile_redirect_uri_override": "Mobile redirect URI override",
-    "oauth_mobile_redirect_uri_override_description": "Enable when OAuth provider does not allow a mobile URI, like '{callback}'",
+    "add_exclusion_pattern_description": "제외 규칙을 추가합니다. *, **, ? 와 같은 glob 패턴을 사용할 수 있습니다. 예: \"Raw\" 폴더 내 모든 파일 제외 → \"**/Raw/**\", .tif 파일 제외 → \"**/*.tif\", 절대 경로 제외 → \"/path/to/ignore/**\"",
+    "asset_offline_description": "이 외부 라이브러리 항목은 디스크에서 찾을 수 없어 휴지통으로 이동되었습니다. 파일이 라이브러리 내에서 이동된 경우, 타임라인에서 새 항목을 확인하세요. 복원을 위해 Immich가 아래 경로에 접근할 수 있어야 하며, 라이브러리 스캔이 필요합니다.",
+    "authentication_settings": "인증 설정",
+    "authentication_settings_description": "비밀번호, OAuth 등 인증 설정을 관리합니다.",
+    "authentication_settings_disable_all": "모든 로그인 수단을 비활성화하시겠습니까? 로그인 기능이 완전히 중단됩니다.",
+    "authentication_settings_reenable": "다시 활성화하려면 <link>서버 명령어</link>를 사용하세요.",
+    "background_task_job": "백그라운드 작업",
+    "backup_database": "데이터베이스 덤프 생성",
+    "backup_database_enable_description": "데이터베이스 덤프 기능을 활성화합니다.",
+    "backup_keep_last_amount": "보관할 이전 덤프 수",
+    "backup_settings": "데이터베이스 덤프 설정",
+    "backup_settings_description": "데이터베이스 덤프 설정을 관리합니다. 참고: 이 작업은 상태를 모니터링하지 않으며 실패 시 알림이 없습니다.",
```

```suggestion
    "backup_settings_description": "데이터베이스 덤프 설정을 관리합니다. 참고: 이 작업은 상태를 모니터링하지 않으며 실패 시 알림이 발생하지 않습니다.",
```
I think “알림이 발생하지 않습니다” sounds a bit more natural than “알림이 없습니다”.